frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx

```diff
@@ -208,7 +223,7 @@ export const BackingStorage: React.FC<BackingStorageProps> = ({
         <Radio
           label={t('ceph-storage-plugin~Connect an external storage platform')}
           description={t(
-            'ceph-storage-plugin~Can be used to connect an external storage platform to OpenShift Data Foundation.',
+            'ceph-storage-plugin~OpenShift Data Foundation will create a dedicated storage class.',
```

Comment:
```suggestion
            'ceph-storage-plugin~OpenShift Data Foundation will create a dedicated StorageClass.',
```

Reply:
cc @rebeccaalpert     ^^

Do we need to do make that change here ?

Reply:
@spadgett - Do you have feelings on this?

Reply:
I would use StorageClass here to be consistent.

Reply:
@afreen23 so we want to do it for all three radio-buttons (both button's label and description)

Reply:
yes @SanjalKatiyar . Lets do for those as well when we have acks from console.